### PR TITLE
Don't invalidate caches on the Mac, and warn users that they need to restart after video download

### LINKED ISSUE #4483: Sidebar is not updating downloaded videos
While testing the current 0.15 MAC develop installer, I noticed that after having downloaded videos as the admin user then going back to the side bar the videos were not listed. I then logged out of admin and logged in as a student still the videos which where downloaded and reflected as downloaded in the video tree where not available to either user using the sidebar.

### PR DESCRIPTION
**Pull request by rtibbles**
edit by @cpauya: fixes #4483

### CODE REVIEW

**Comment by rtibbles** [comment]
Fixes #4483

**Comment by amodia** [comment]
@rtibbles & @cpauya I tested this PR on OS X Yosemite, and the warning message does not show after the video download finished.

screen shot 2015-09-28 at 3 57 17 pm

screen shot 2015-09-28 at 3 57 39 pm

I got this browser console error when downloading videos.
screen shot 2015-09-28 at 4 00 42 pm

**Comment by cpauya** [comment]
@amodia The "warning message" is supposed to tell the Mac user to restart the server after downloading a video and NOT "the warning message does not show after the video download finished." as you expected.

Did it show the warning message?

**Comment by djallado** [comment]
I build mac installer based on this PR and I can see the message. 
Here's the screenshot:

screen shot 2015-09-28 at 5 58 37 pm

The message appear when I click on `Video` tab.

**Comment by cpauya** [comment]
Ok, got the message running here.

screenshot 2015-09-28 18 46 53

Merging this now.

**Comment by amodia** [comment]
Yeah, It's also working on me. I just removed the" KALITE_PYTHON" environment and setup new virtualenv.
screen shot 2015-09-28 at 7 17 57 pm

**Comment by cpauya** [comment]
Oh, there's a Reopen button!  Damn I already created #4518. :(

File: kalite/updates/views.py [hunk 1]
```diff
@@ -32,7 +34,7 @@ def update_videos(request, max_to_show=4):
     if getattr(settings, 'USING_RASPBERRY_PI', False):
         messages.warning(request, _('For low-powered devices like the Raspberry Pi, please download less than 25 videos at a time.'))
 
-    if DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP:
+    if DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP or sys.platform == 'darwin':
```
Comment (benjaoming) [inline kalite/updates/views.py]: Sorry, didn't see this before now.

Why not just add a custom settings module for the mac installer and have `DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP = True` ?

Comment (rtibbles) [inline kalite/updates/views.py]: I refer you to the title of the commit.

Comment (benjaoming) [inline kalite/updates/views.py]: Well then :/

Tracking here: https://github.com/learningequality/installers/issues/206

Comment (rtibbles) [inline kalite/updates/views.py]: This all gets blown away in 0.16.

**Pull request closed without merging**